src/Microsoft.Data.SqlClient/tests/ManualTests/Microsoft.Data.SqlClient.ManualTesting.Tests.csproj

```
@@ -10,6 +10,7 @@
     <Platforms>AnyCPU;x86;x64</Platforms>
     <DefineConstants Condition="'$(TargetGroup)'=='netcoreapp'">$(DefineConstants);NETCOREAPP</DefineConstants>
     <DefineConstants Condition="'$(TargetGroup)'=='netfx'">$(DefineConstants);NETFRAMEWORK</DefineConstants>
+    <DefineConstants Condition="'$(TargetGroup)' == 'netcoreapp' AND (!$(TargetFramework.StartsWith('netcoreapp')) OR (!$(TargetNetCoreVersion.StartsWith('netcoreapp2.1')) AND !$(TargetNetCoreVersion.StartsWith('netcoreapp3.1'))))">$(DefineConstants);NET50_OR_LATER</DefineConstants>
```

```suggestion
    <DefineConstants Condition="'$(TargetGroup)' == 'netcoreapp' AND !$(TargetFramework.StartsWith('netcoreapp'))">$(DefineConstants);NET50_OR_LATER</DefineConstants>
```
It should be simplified to above.

Please also ensure "TargetGroup" is set to 'netcoreapp' for .NET 5.0+ in **both Manual Tests and Functional Tests.**
I'm curious to see if pipelines actually test for **TargetGroup = netcoreapp** when targetFramework is **"net5.0"**?

Best solution seems to me:
```
    <TargetGroup Condition="$(TargetFramework.StartsWith('net4'))">netfx</TargetGroup>
    <TargetGroup Condition="$(TargetGroup) == ''">netcoreapp</TargetGroup>
```

---

Done as the suggestion.